CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ if(WIN32)
     set(SFML_DIR "${SFML_HOME}/lib/cmake/SFML")
 endif()
 
+if(NOT EXISTS "${CMAKE_SOURCE_DIR}/modules/fft-real/ffft")
+    message(FATAL_ERROR "The submodule fft-real was not found. If you use git, run this command \"git submodule update --init --recursive\". If you do not use git, please download the module from https://github.com/andywang0607/fft-real and place it in the \"modules/\" folder")
```

Comment:
@BenPyton this looks nice, but the forward slash will likely feel alian to users on Windows (so I'd maye drop it) and the end seems to be missing a full stop, would at least look more natural with a full stop. Just an idea.

Reply:
@hartwork I presume that users willing to compile this software have some little background on programing. I am a Windows user and I am used to read paths with backslashes as well as forward slashes.

However you're right, I will do some checks to change the path displayed depending on the platform.

Also you're right for the full stop at the end, I'll add it.